src/ResultWriter/EnergyOutput.cpp

```diff
@@ -698,7 +698,8 @@ void EnergyOutput::writeHeader() { out << "time,variable,measurement" << std::en
 
 void EnergyOutput::writeEnergies(double time) {
   if (shouldComputeVolumeEnergies()) {
-    out << time << ",gravitational_energy," << energiesStorage.gravitationalEnergy() << "\n"
+    out << std::setprecision(10) << time << ",gravitational_energy,"
```

Comment:
The `setprecision` here is covered by `Output/EnergyTerminalPrecision` by now.

But we can increase the default (currently 6) for it, of course.